### DISCUSSION TITLE
Navigating to a WikiLink

### DISCUSSION THREAD

**Discussion by Georift** [gsantner/markor#1917] [Q&A] [answered]
Hey thanks for an amazing app! Fits beautifully into my existing sync-thing / note taking!

I'm using [Foam](https://github.com/foambubble/foam) on my desktop to take notes. I believe it uses WikiLinks syntax? (eg. [[PageName]])

I've noticed when I switch viewing a markdown file with these I can navigate to them perfectly! However I was wondering if there was a way to navigate without first switching to the view screen?

I'd imagine something like the link highlighting +  the Goto link button? Like how markdown links are handled in edit mode now could be nice?

Happy to take a crack at implementing this if there is support from the community.

**Comment by harshad1** [comment, marked as answer]
I think this would be useful for me too. I can help if you start a PR
